test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -2165,11 +2086,14 @@ def validate(self):
                                 code='invalid-module-deprecation-source',
                                 msg=('The deprecation version for a module must be added in this collection')
                             )
-                            # Treat the module as not to be removed:
-                            raise ValueError('')
-                        removed_in = self.StrictVersion(str(version))
+
+                            end_of_deprecation_should_be_removed_only = False
```

Comment:
The way you've changed this code, it now no longer works: if `end_of_deprecation_should_be_removed_only` is set to` False` here without raising an exception, the `else:` after the `except` will get executed, but that code assumes that `removed_in` has been set. So if this happens, validate-module will now crash.